rflx/pyrflx/typevalue.py

```diff
@@ -221,3 +330,502 @@ def accepted_type(self) -> type:
     @property
     def literals(self) -> Mapping[Name, Expr]:
         return {}
+
+
+class ArrayValue(CompositeValue):
+
+    _value: List[TypeValue]
+
+    def __init__(self, vtype: Array) -> None:
+        super().__init__(vtype)
+        self._element_type = vtype.element_type
+        self._is_message_array = isinstance(self._element_type, Message)
+        self._value = []
```

Comment:
Same as above, identifiers only have to declared directly in the package if they're not declared in `__init__`. Unless we want to keep it for consistency? @treiher

Reply:
You would have to add the type in `__init__` to make it equivalent: `self._value: List[TypeValue]`. Otherwise it would be `List[Any]`.

For consistency reasons I would keep it as it is.

Reply:
Why is it removed now?

Reply:
`_value: List[TypeValue]` is still in line 334. I didn't edit this section.

Reply:
Oh, my fault. I compared a wrong version locally. It is hard to track the changes after force-push.